fe/app/components/EventGrid/EventGrid.tsx

```diff
@@ -0,0 +1,99 @@
+import RoundArrowButton from "@components/RoundArrowButton/RoundArrowButton";
+import { useCallback, useEffect, useState } from "react";
+import EventElement from "../EventElement/EventElement";
+import { EventElementProps } from "../EventElement/EventElement.models";
+import { useTranslations } from "next-intl";
+
+const EventGrid = ({ events }: { events: EventElementProps[] }) => {
```

Comment:
I know it has only one prop but shouldn't we still have a `models.ts` file to go along with the standard we are using on the rest of the components?